exercises/settings.gradle

```diff
@@ -33,6 +33,7 @@ include 'robot-name'
 include 'roman-numerals'
 include 'prime-factors'
 include 'scrabble-score'
+include 'series'
```

Comment:
+1 for keeping with the alphabetical ordering.  It's little details like this that keep the place tidy and encourage others to do so. :)